test-framework/common/src/main/java/io/quarkus/test/common/QuarkusTestResourceLifecycleManager.java

```diff
@@ -31,6 +31,18 @@
      */
     void stop();
 
+    /**
+     * Arguments passed to the lifecycle manager before it starts
+     * These arguments are taken from {@code QuarkusTestResource#args()}
```

Comment:
Nitpick: `{@code QuarkusTestResource#initArgs()}`

Reply:
Ah darn... I thought I had changed all of those :)